sql/database.sql

```diff
@@ -5255,18 +5295,47 @@ INSERT INTO list_options (list_id, option_id, codes,title, seq) VALUES ('physici
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`, `edit_options`, `timestamp`, `last_updated`) VALUES ('Industry','531110','Lessors of Residential Buildings',110,0,0,'',NULL,'531110.007615',0,0,1,'',1,'2025-09-17 02:02:03','2025-09-16 22:02:03');
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`, `edit_options`, `timestamp`, `last_updated`) VALUES ('Industry','484121','General Freight Trucking, Long-Distance',120,0,0,'',NULL,'484121.007193',0,0,1,'',1,'2025-09-17 02:02:03','2025-09-16 22:02:03');
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`, `edit_options`, `timestamp`, `last_updated`) VALUES ('Industry','812111','Barber Shops',130,0,0,'',NULL,'812111.011099',0,0,1,'',1,'2025-09-17 02:02:03','2025-09-16 22:02:03');
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`, `edit_options`, `timestamp`, `last_updated`) VALUES ('Industry','522110','Commercial Banking',140,0,0,'',NULL,'522110.007773',0,0,1,'',1,'2025-09-17 02:02:03','2025-09-16 22:02:03');
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`, `edit_options`, `timestamp`, `last_updated`) VALUES ('Industry','999999','Unemployed',150,0,0,'',NULL,'999999',0,0,1,'',1,'2025-09-17 02:02:03','2025-09-16 22:02:03');
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`, `edit_options`, `timestamp`, `last_updated`) VALUES ('Industry','UNKNOWN','Unknown',160,0,0,'',NULL,'UNKNOWN',0,0,1,'',1,'2025-09-17 02:02:03','2025-09-16 22:02:03');
 -- Occupation
-
-INSERT INTO `list_options` (`list_id`, `option_id`, `title`) VALUES('lists','Occupation','Occupation');
-INSERT INTO list_options ( list_id, option_id, title, seq ) VALUES ('Occupation', 'lawyer', 'Lawyer', 10);
-INSERT INTO list_options ( list_id, option_id, title, seq ) VALUES ('Occupation', 'engineer', 'Engineer', 20);
-INSERT INTO list_options ( list_id, option_id, title, seq ) VALUES ('Occupation', 'site_worker', 'Site Worker', 30);
```

Comment:
Wouldn't overwrite a current list. Would instead create a new list with different name/list_id. Note this will also not work in the upgrade script.

Reply:
Not overwriting because at this point the list doesn't exist and maintains legacy for access of occupation and industry in Employer group.
While I made a late change in upgrade to prevent list update I have since fixed and new list items will append to existing.
It's that or delete list the add new.
New list is not appropriate because these list are old legacy in Employer plus tied to ccda and fhir so list name integrity is important. But thanks you caught my boo boo when I late added meta tags in upgrade.
Btw: JSON was Goobers fault!:)